tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3541,3 +3542,63 @@ def combined_non_max_suppression(boxes,
     return gen_image_ops.combined_non_max_suppression(
         boxes, scores, max_output_size_per_class, max_total_size, iou_threshold,
         score_threshold, pad_per_class)
+  
+  
+@tf_export('image.median_filtering_2D')
+def median_filtering_2D(tf_img,filter_shapex=3,filter_shapey=3):
```

Comment:
The argument names should probably be (input, filter_shape) where filter_shape is a 2-element tuple. The function should be named median_filter_2d to match conv_2d and other things in the tf namespace.